pkg/util/winutil/iisconfig/tagtree.go

```diff
@@ -0,0 +1,157 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+//go:build windows
+
+package iisconfig
+
+import (
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"golang.org/x/sys/windows/registry"
+)
+
```

Comment:
Can you add a comment explaining how pathTree(Entry) works. It is not entirely clear